src/commands/map.ts

```diff
@@ -117,8 +117,16 @@ export default class Map extends Command {
       saved ? 'Boilerplate code prepared.' : 'Boilerplate code already exists.'
     );
 
+    ux.start('Setting up local project');
     // TODO: install dependencies
     await prepareJsProject(undefined, undefined, { logger });
+
+    ux.succeed(
+      // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+      `Local project set up. You can now install defined dependencies and run \`superface execute ${
```

Comment:
question: Will we also provider instructions for installing dependencies per target language?